asv/.github/workflows/flake8.yml

```diff
@@ -0,0 +1,24 @@
+
+name: flake8
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
This looks good, but for linting probably linting with the latest python version is enough. When linting the asv code is not executed, so we're just saying here to run flake8 with those different versions of python, which doesn't add a lot of value. For testing this would make total sense.

Reply:
Great analogy, in case of feature new versions, don't have to come back to edit it.